packages/select/src/FilterMultiSelect/components/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useId } from "react"
+import { InputSearch } from "@kaizen/draft-form"
+import { useSelectionContext } from "../../provider"
+
+export interface SearchInputProps {
+  label: string
```

Comment:
I wonder if we should make this optional and provide a default, otherwise we might get people putting some strange label in here. Maybe we could default to "Filter options by search query".